docroot/themes/humsci/humsci_basic/src/scss/components/_pager.scss

```diff
@@ -0,0 +1,91 @@
+.pager {
+  &__items {
+    list-style: none;
+    margin: hb-calculate-rems(10px) 0 hb-calculate-rems(20px);
+    padding: 0;
+    text-align: center;
+  }
+
+  &__item {
+    display: inline;
+    font-weight: hb-theme-font-weight(semibold);
+    margin: hb-calculate-rems(10px);
+
+    &:first-of-type {
+      margin-left: 0;
+    }
+
+    &:last-of-type {
+      margin-right: 0;
+    }
+
+    a {
+      @include hb-colorful {
+        @include hb-global-color('color', 'gray-dark');
+      }
+
+      display: inline-block;
+      padding: hb-calculate-rems(6px);
+      border-radius: 50%;
+      width: hb-calculate-rems(31px);
```

Comment:
This should be `min-width`, the "Show More" button in the AJAX pagination also uses these styles and the set width has the text overflowing the button.